test/python/test_eval_json.py

```diff
@@ -0,0 +1,92 @@
+"""Test the evaluation of JSON results."""
+
+from __future__ import annotations
+
+import pytest
+
+from mqt.core.evaluation import compare, flatten_dict, higher_better
+
+path_base = "test/python/example_baseline_results.json"
+path_feature = "test/python/example_feature_results.json"
```

Comment:
this will not work in general if I am not mistaken as it is dependent on the directory in which you call pytest.
a simple
```
cd test
pytest
```
probably already fails. There are ways to mitigate this that involve `importlib.resources`.
A quick google search should bring up a few results.